### PR TITLE
Update quest instructions for QuestPatcher 2.

### PR DESCRIPTION
**Pull request by Lauriethefish**
QuestPatcher 2 will soon be out, and hopefully it'll make modding easier.
This PR updates all of the instructions to be relevant to QuestPatcher 2.

Separate sections of the quest install guide have also been created, based on if you're on windows, mac or linux.

The cosmetic installations using SideQuest have been swapped to using the QuestPatcher cosmetics menu.
Some warnings that are no longer an issue have been removed, since QuestPatcher now has better warnings.

Please review the changes, some could have issues as this is an early draft.

### CODE REVIEW

File: docs/user-guide/quest/beginners-guide.md [hunk 1]
```diff
@@ -5,83 +5,45 @@
...
 
-> These instructions are for **Windows** only at the moment. If you want to try a Linux build of QuestPatcher, you can find it [here as a .zip file](https://github.com/Lauriethefish/QuestPatcher/releases/latest).  
-> Note: If Windows alerts you that the file is unsafe/at risk, choose to keep/run anyway. This is a false-positive.
+### Windows
 
-Currently the only recommended way to install mods is **QuestPatcher**. Download **QuestPatcher** from [Lauriethefish's GitHub](https://github.com/Lauriethefish/QuestPatcher/releases/latest). Select the **Assets** dropdown and download the .exe file named `windows-installer.exe`. Double-click the executable to begin installing, and follow the instructions.
+> The only version of Windows officially supported is Windows 10.
+> Other versions might work, but we cannot guarantee full functionality.
 
+1. Select the assets dropdown on github, and click `QuestPatcher-windows.exe`, then wait for the download to finish.
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:17]: Proper names are preferred for consistency, in this case please use *GitHub*

File: docs/user-guide/quest/beginners-guide.md [hunk 2]
```diff
@@ -5,83 +5,45 @@
...
+2. Open your downloads folder, and double click the installer.
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:18]: *Downloads*

File: docs/user-guide/quest/beginners-guide.md [hunk 3]
```diff
@@ -5,83 +5,45 @@
...
+3. It's possible that windows will prevent you from opening the installer, as it is from an unknown source. Just press `More Info` and `Run Anyway`.
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:19]: *Windows*: also preferred if you can put an image of SmartScreen here :D

File: docs/user-guide/quest/beginners-guide.md [hunk 4]
```diff
@@ -5,83 +5,45 @@
...
+4. Select `Install for all users` or `Install for me only` and press `Yes` if an admin prompt comes up.
+5. Accept the license agreement, press next, press next again, then press install.
+6. Wait until the install finishes, then press next, and then finish to open QuestPatcher.
 
-![Preview](/docs/files/questpatcherprocessing.png)
 
-!> If you haven't modded a game or sideloaded apps to your headset before, you'll need to get **SideQuest** in addition to QuestPatcher. [**Follow this link**](https://sidequestvr.com/setup-howto) to learn how to set it up. After you've finished, you should have setup your Quest with developer mode on, and SideQuest showing a green status indicator at the top left when it is plugged in.
+### Mac OS
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:25]: *macOS*

File: docs/user-guide/quest/beginners-guide.md [hunk 5]
```diff
@@ -5,83 +5,45 @@
...
 
-## Install Instructions
+!> QuestPatcher should automatically download Java. If you're on an older version of Mac OS, Java might be detected as already installed when it isn't. If you are having trouble patching, manually download Java [here](https://www.java.com/en/).
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:27]: *macOS*

File: docs/user-guide/quest/beginners-guide.md [hunk 6]
```diff
@@ -5,83 +5,45 @@
...
+5. Open your Applications folder and right click QuestPatcher. **__Make sure you hold control__**, then click open (You will only have to do this the first time).
+6. In the unrecognized developer that comes up, click `Open`.
 
-!> **Do NOT close QuestPatcher while it is downloading "platform-tools" if you don't have them already.**
+### Linux
 
-After you've downloaded and installed QuestPatcher, open it.
-By default, QuestPatcher should automatically detect your Gorilla Tag app.
+> QuestPatcher is compiled for [Ubuntu](https://ubuntu.com/) Linux.
+> Other linux distributions may work, but you may encounter issues.
```
Comment (burritosoftware) [inline docs/user-guide/quest/beginners-guide.md:39]: *Linux*

**Review by burritosoftware — CHANGES_REQUESTED** [review]
Please make sure to capitalize names properly for consistency and maybe clarify prompts like Windows SmartScreen and the macOS gatekeeper with screenshots so it's better looking and easier to understand :D

File: docs/user-guide/quest/beginners-guide.md (hunk 6 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md]: Fixed

File: docs/user-guide/quest/beginners-guide.md (hunk 5 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md]: Fixed

File: docs/user-guide/quest/beginners-guide.md (hunk 4 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md]: Fixed

File: docs/user-guide/quest/beginners-guide.md (hunk 3 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md:19]: Will add smartscreen image, if I can convince my PC to show it

File: docs/user-guide/quest/beginners-guide.md (hunk 2 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md:18]: Fixed

File: docs/user-guide/quest/beginners-guide.md (hunk 1 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md]: Fixed

File: docs/user-guide/quest/beginners-guide.md (hunk 3 above)
Comment (Lauriethefish) [reply docs/user-guide/quest/beginners-guide.md]: Added smartscreen image.

**Review by burritosoftware — CHANGES_REQUESTED** [review]
Please add a macOS gatekeeper screenshot, thanks :)

**Comment by Lauriethefish** [comment]
Done

**Review by burritosoftware — APPROVED** [review]
Looks good to me, will merge when released

**Pull request merged**